tests/test_external_plugins.py

```diff
@@ -87,6 +122,14 @@ def load_test_plugin(plugin_filename: str, disable_caching: bool = False, *args,
     return plugin
 
 
+def delete_installed_plugins():
+    for plugin_file in PLUGIN_FILES_TO_DELETE:
+        shutil.rmtree(plugin_file)
+
+
+atexit.register(delete_installed_plugins)
```

Comment:
Looks like the more "pytest" way of doing this is to make this a [fixture](https://docs.pytest.org/en/6.2.x/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request) in conftest.py with` scope="session"`  and `autouse=true` then pass it to every test. They recommend using [yield for clean up](https://docs.pytest.org/en/6.2.x/fixture.html#yield-fixtures-recommended).

With this I think you can install the plugin file copying/deletion as different fixtures.